packages/modules/playwright/src/playwright-container.ts

```diff
@@ -0,0 +1,202 @@
+import { copyFile } from "fs/promises";
+import path from "path";
+import tar from "tar-fs";
+import tmp from "tmp";
+import { existsSync, mkdirSync } from "fs";
+
+import {
+  AbstractStartedContainer,
+  AbstractStoppedContainer,
+  GenericContainer,
+  StartedTestContainer,
+  StopOptions,
+  StoppedTestContainer,
+  log,
+} from "testcontainers";
+
+const CONTAINER_WORKING_DIRECTORY = "/playwright";
+const DEFAULT_JSON_REPORTER_FILE = "results.json";
+const DEFAULT_HTML_REPORTER_OUTPUT_DIRECTORY = "/test-reports";
+const DEFAULT_HTML_REPORTER_FILE = "index.html";
+const DEFAULT_BLOB_REPORTER_FILE = "report.zip";
+const DEFAULT_BLOB_REPORTER_OUTPUT_DIRECTORY = "/blob-report";
+const DEFAULT_JUNIT_REPORTER_FILE = "results.xml";
+
+const EXPORTABLE_REPORTER_TYPE = {
+  JSON: "json",
+  HTML: "html",
+  BLOB: "blob",
+  JUNIT: "junit",
+} as const;
+
+type ExportableReporterType = (typeof EXPORTABLE_REPORTER_TYPE)[keyof typeof EXPORTABLE_REPORTER_TYPE];
+
+type ContainerReporter = {
+  type: ExportableReporterType;
+  outputFile?: string;
+  outputDir?: string;
+  fileName?: string;
+};
+
+export class PlaywrightContainer extends GenericContainer {
+  constructor(
+    image = "mcr.microsoft.com/playwright:latest",
```

Comment:
Probably not use latest, set explicitly the last version number

Reply:
refactor here https://github.com/testcontainers/testcontainers-node/pull/726/commits/b6930c480ce6836e15c7204eb7277432f84adeab